explainaboard/processors/language_modeling.py

```diff
@@ -125,34 +114,57 @@ def default_analyses(self) -> list[AnalysisLevel]:
                 func=lambda info, x, c, stat: stat['vocab'].get(c.text, 0.0),
             ),
         }
-        tok_continuous_features = [
-            k
-            for k, v in tok_features.items()
-            if ('float' in unwrap(v.dtype))
-            if k != 'tok_log_prob'
-        ]
-        tok_analyses: list[BucketAnalysis] = [
-            BucketAnalysis(
-                description=tok_features[x].description, feature=x, method="continuous"
-            )
-            for x in tok_continuous_features
-        ]
 
         return [
             AnalysisLevel(
                 name='example',
                 features=examp_features,
                 metric_configs=self.default_metrics(level='example'),
-                analyses=cast(List[Analysis], examp_analyses),
             ),
             AnalysisLevel(
-                name='tok',
+                name='token',
                 features=tok_features,
-                metric_configs=self.default_metrics(level='tok'),
-                analyses=cast(List[Analysis], tok_analyses),
+                metric_configs=self.default_metrics(level='token'),
             ),
         ]
 
+    def default_analyses(self) -> list[Analysis]:
+        analyses: list[Analysis] = []
+        examp_features = self.default_analysis_levels()[0].features
+        examp_continuous_features = [
+            k for k, v in examp_features.items() if ('float' in unwrap(v.dtype))
+        ]
+        analyses.extend(
+            [
+                BucketAnalysis(
+                    level="example",
+                    description=examp_features[x].description,
+                    feature=x,
+                    method="continuous",
+                )
+                for x in examp_continuous_features
+            ]
+        )
+        tok_features = self.default_analysis_levels()[1].features
+        tok_continuous_features = [
+            k
+            for k, v in tok_features.items()
+            if ('float' in unwrap(v.dtype))
+            if k != 'tok_log_prob'
+        ]
+        analyses.extend(
+            [
+                BucketAnalysis(
+                    level="token",
+                    description=tok_features[x].description,
+                    feature=x,
+                    method="continuous",
+                )
+                for x in tok_continuous_features
+            ]
+        )
```

Comment:
Several times the `Analysis` object is generated via multiple comprehension. Using a usual for-loop would be much simpler and faster. (same for others)
```suggestion
        for feature_name, feature_value in self.default_analysis_levels()[1].features:
            if feature_name == "tok_log_prob":
                break
            if "float" in unwrap(v.dtype):
                break
            analyses.append(
                BucketAnalysis(
                    level="token",
                    descritpion=feature_value.description,
                    method="continuous",
                )
            )
```